samples/openapi3/client/petstore/dart-dio-next/petstore_client_lib_fake/doc/PetApi.md

```diff
@@ -361,7 +361,7 @@ Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
  **petId** | **int**| ID of pet to update | 
  **additionalMetadata** | **String**| Additional data to pass to server | [optional] 
- **file** | **Uint8List**| file to upload | [optional] 
```

Comment:
this isnt backwards compatible, is it?

Reply:
No it isn't, but the dart-dio-next generator is marked experimental. And the fix is quick while adding support for file names.